scripts/BuildPackagesFromSigningDirectory.ps1

```diff
@@ -0,0 +1,57 @@
+<#
+.SYNOPSIS
+    Package the SARIF SDK from a custom signing directory.
+.DESCRIPTION
+    Builds the SARIF SDK NuGet Packages from a custom signing directory after
+    they have been signed.
+.PARAMETER Configuration
+    The build configuration: Release or Debug. Default=Release
+#>
+
+[CmdletBinding()]
+param(
+    [string]
+    [ValidateSet("Debug", "Release")]
+    $Configuration="Release"
+)
+
+Import-Module -Force $PSScriptRoot\ScriptUtilities.psm1
+Import-Module -Force $PSScriptRoot\Projects.psm1
+
+# Copy signed binaries back into the normal directory structure.
+function CopyFromSigningDirectory {
```

Comment:
>CopyFromSigningDirectory [](start = 9, length = 24)

May as well use that standard PowerShell form `verb-noun` to get `Copy-FromSigningDirectory`.